extra/2-piping.js

```diff
@@ -16,26 +16,37 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(pipe1,pipe2) {
+ var total = pipe1 + pipe2;
+ return total;
 }
 
-function multiply() {
-
+function multiply(num1,num2) {
+ var multiplyNum = num1 * num2;
+ return multiplyNum;
 }
 
-function format() {
-
+function format(number) {
+var message = "£" + number;
+return message;
 }
 
 const startingValue = 2;
 
+function digit(startingValue) {
+  let calculation = "£" + (startingValue + 10) * 2; return calculation;
+}
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+// This code is difficult to read and look messy.
+let badCode = digit(startingValue);
 
 /* BETTER PRACTICE */
+function goodDigit(startingValue) {
```

Comment:
This is not much different to your badCode. You should write something that calls the 3 functions in one line. So  start off by calling the add function with the startingValue and 2, then wrap that in the multiply function, then wrap that in the format function.